config/sync/webform.webform.contact.yml

```diff
@@ -1,6 +1,6 @@
 uuid: cc3f4d2f-b0ab-4121-a21d-9cc8a17aa139
 langcode: en
-status: open
+status: closed
```

Comment:
Intended to be closed?

Reply:
Yes - it's a security vulnerability - as per PR description